### PR TITLE
:sparkles: Add CNCF Nantes meetup details

### PR DESCRIPTION
**Pull request by ebriand**

### CODE REVIEW

**Review by geromegrignon — CHANGES_REQUESTED** [review]
Thanks for your contribution @ebriand.

Can you update the CNCF link of the [README](https://github.com/nantes-tech-communities/list-communities/blob/main/README.md) file to `./cncf-nantes`?

**Comment by jtanguy** [comment]
Note: the trailing slash is needed

**Comment by ebriand** [comment]
Thanks for the review @geromegrignon and @jtanguy 👌  It is changed 👍

File: cncf-nantes/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# CNCF Nantes 
+
+|                                |     |
+| ------------------------------ | --- |
+|                | ![logo](logo.jpeg)   
+| ✉️ Qui contacter ?             | [Gaëlle Acas](https://twitter.com/gaelleacas)<br/>[Mickaël Alibert](https://twitter.com/alibert_m)</br>[Eric Briand](https://twitter.com/eric_briand)<br/>[Lucas Dupuy](https://twitter.com/louckousse)<br/>[Vincent Poilvert](https://twitter.com/vincentpoilvert) |
```
Comment (jtanguy) [inline cncf-nantes/README.md:6]:
```suggestion
| ✉️ Qui contacter ?             | [Gaëlle Acas](https://twitter.com/gaelleacas)<br/>[Mickaël Alibert](https://twitter.com/alibert_m)<br/>[Eric Briand](https://twitter.com/eric_briand)<br/>[Lucas Dupuy](https://twitter.com/louckousse)<br/>[Vincent Poilvert](https://twitter.com/vincentpoilvert) |
```

**Review by jtanguy — CHANGES_REQUESTED** [review]

**Pull request merged**